app/tests/spec/views/base.js

```diff
@@ -219,7 +215,7 @@ define(function (require, exports, module) {
           sessionToken: 'abc123',
           uid: 'uid'
         });
-        sinon.stub(fxaClient, 'sessionStatus', function () {
+        sinon.stub(account, 'isSignedIn', function () {
```

Comment:
Just wondering: If we're stubbing this, should we also assert that it is called as we expect?